modules/shortcodes/bandcamp.php

```diff
@@ -93,97 +130,97 @@ function shortcode_handler_bandcamp( $atts ) {
 		array_push( $argparts, "t={$attributes['t']}" );
 	}
 
-	if ( $attributes['notracklist'] == "true" ) {
-		array_push( $argparts, "notracklist=true" );
+	if ( $attributes['notracklist'] == 'true' ) {
+		array_push( $argparts, 'notracklist=true' );
 	}
 
 	// 'tracklist' arg deprecates 'notracklist=true' to be less weird.  note, behavior
 	// if both are specified is undefined
 	switch ( $attributes['tracklist'] ) {
-		case "false":
-		case "none":
-			array_push( $argparts, "tracklist=false" );
+		case 'false':
+		case 'none':
+			array_push( $argparts, 'tracklist=false' );
 			break;
 	}
 
 	switch ( $attributes['artwork'] ) {
-		case "false":
-		case "none":
-		case "small":
-			array_push( $argparts, "artwork=" . $attributes['artwork'] );
+		case 'false':
+		case 'none':
+		case 'small':
+			array_push( $argparts, 'artwork=' . $attributes['artwork'] );
 			break;
 	}
 
-	if ( $attributes['minimal'] == "true" ) {
-		array_push( $argparts, "minimal=true" );
+	if ( $attributes['minimal'] == 'true' ) {
+		array_push( $argparts, 'minimal=true' );
 	}
 
-	if ( isset( $attributes['theme'] ) && preg_match( "|^[a-zA-Z_]+$|", $attributes['theme'] ) ) {
+	if ( isset( $attributes['theme'] ) && preg_match( '|^[a-zA-Z_]+$|', $attributes['theme'] ) ) {
 		array_push( $argparts, "theme={$attributes['theme']}" );
 	}
 
 	// param 'tracks' is signed digest param 'esig'
-	if ( isset( $attributes['tracks'] ) && preg_match( "|^[0-9\,]+$|", $attributes['tracks'] ) ) {
-		if ( isset( $attributes['esig'] ) && preg_match( "|^[0-9A-Fa-f]+$|", $attributes['esig'] ) ) {
+	if ( isset( $attributes['tracks'] ) && preg_match( '|^[0-9\,]+$|', $attributes['tracks'] ) ) {
+		if ( isset( $attributes['esig'] ) && preg_match( '|^[0-9A-Fa-f]+$|', $attributes['esig'] ) ) {
 			array_push( $argparts, "tracks={$attributes['tracks']}" );
 			array_push( $argparts, "esig={$attributes['esig']}" );
 		}
 	}
 
 	if ( $isVideo ) {
-		$url        = "//bandcamp.com/VideoEmbed?" . join( '&', $argparts );
+		$url        = '//bandcamp.com/VideoEmbed?' . join( '&', $argparts );
 		$extraAttrs = " mozallowfullscreen='1' webkitallowfullscreen='1' allowfullscreen='1'";
 	} else {
-		$url        = "//bandcamp.com/EmbeddedPlayer/v=2/" . join( '/', $argparts ) . '/';
+		$url        = '//bandcamp.com/EmbeddedPlayer/v=2/' . join( '/', $argparts ) . '/';
 		$extraAttrs = '';
 	}
 
-	return "<iframe width='" . esc_attr( $width ) . "' height='" . esc_attr( $height ) . "' style='position: relative; display: block; width: " . esc_attr( $csswidth ) . "; height: " . esc_attr( $cssheight ) . ";' src='" . esc_url( $url ) . "' allowtransparency='true' frameborder='0'" . $extraAttrs . "></iframe>";
+	return "<iframe width='" . esc_attr( $width ) . "' height='" . esc_attr( $height ) . "' style='position: relative; display: block; width: " . esc_attr( $csswidth ) . '; height: ' . esc_attr( $cssheight ) . ";' src='" . esc_url( $url ) . "' allowtransparency='true' frameborder='0'" . $extraAttrs . '></iframe>';
```

Comment:
Looks like it did not catch the double quotes at the beginning of the statement.

Reply:
I think that's intentional since it is jumping back and forth with single/double quotes since the html is using double.